WebUI/src/assets/js/store/globalSetup.ts

```diff
@@ -67,18 +70,20 @@ export const useGlobalSetup = defineStore("globalSetup", () => {
 
     let envType = "";
 
-    type globalSetupState = "running" | "verifyBackend" | "manageInstallations" | "loading" | "failed"
-
-    const loadingState: Ref<globalSetupState> = ref("verifyBackend");
-
+    const loadingState = ref<GlobalSetupState>("verifyBackend");
     const errorMessage = ref("");
-
     const hdPersistentConfirmation = ref(localStorage.getItem("HdPersistentConfirmation") === "true");
 
     watchEffect(() => {
         localStorage.setItem("HdPersistentConfirmation", hdPersistentConfirmation.value.toString());
     });
 
+    watch([backendServices.allRequiredRunning], (newVal) => {
+        if (newVal) {
+            initSetup();
+        }
```

Comment:
and what do we do with the state if newVal is false? set state to manageInstallations?